source/changenotes.ttl

```diff
@@ -0,0 +1,25 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix : <https://id.kb.se/vocab/>
+
+base <https://id.kb.se/>
+
+</changenote/primarycontribution> a :ChangeCategory ;
+  skos:prefLabel "Change of primary contribution"@en, "Ändring av primärt upphov"@sv .
+
+</changenote/worktitle> a :ChangeCategory ;
+  skos:prefLabel "Change of work title"@en, "Ändring av verkets titel"@sv .
+
+</changenote/primarytitle> a :ChangeCategory ;
+  skos:prefLabel "Change of primary title"@en, "Ändring av huvudtitel"@sv .
```

Comment:
```suggestion
</changenote/instancetitle> a :ChangeCategory ;
  skos:prefLabel "Change of primary title"@en, "Ändring av huvudtitel"@sv .
```
Maybe `changenote/instancetitle` or `changenote/maintitle`

"Title proper" in RDA lingo?
"Huvudtitel" is used @ metadatayrån

(Outside of the scope here there is a bit of confusion regarding the naming of `Title` + `maintitle` vs other tiles)

Reply:
Yes, some clarification (and/or simplification) would be good here. We may rename this once we see practical usage during testing.